src/transformers/generation_utils.py

```diff
@@ -572,14 +572,14 @@ def _get_decoder_start_token_id(self, decoder_start_token_id: int = None, bos_to
     @staticmethod
     def _expand_inputs_for_generation(
         input_ids: torch.LongTensor,
-        expand_size: int = 1,
+        num_return_sequences: int = 1,
```

Comment:
Renamed this arg as it is a private method, it is more readable, and it is useful for name matching (as you'll see below)